examples/connector/python/main.py

```diff
@@ -63,86 +92,85 @@ def Update(self, request, context):
 
         # -- Send UPSERT records
         for t in range(0, 3):
-            operation = connector_sdk_pb2.Operation()
+            operation = connector_sdk_pb2.UpdateResponse()
             val1 = common_pb2.ValueType()
             val1.string = "a-" + str(t)
 
             val2 = common_pb2.ValueType()
             val2.double = t * 0.234
 
             record = connector_sdk_pb2.Record()
-            record.type = common_pb2.OpType.UPSERT
+            record.type = common_pb2.UPSERT
             record.table_name = "table1"
             record.data["a1"].CopyFrom(val1)
             record.data["a2"].CopyFrom(val2)
             state["cursor"] += 1
 
             operation.record.CopyFrom(record)
-            yield connector_sdk_pb2.UpdateResponse(operation=operation)
+            yield connector_sdk_pb2.UpdateResponse(record=record)
 
         # -- Send UPSERT record for table2
-        operation = connector_sdk_pb2.Operation()
+        operation = connector_sdk_pb2.UpdateResponse()
         val1 = common_pb2.ValueType()
         val1.string = "b1"
         val2 = common_pb2.ValueType()
         val2.string = "ben"
         record = connector_sdk_pb2.Record()
-        record.type = common_pb2.OpType.UPSERT
+        record.type = common_pb2.UPSERT
         record.table_name = "table2"
         record.data["b1"].CopyFrom(val1)
         record.data["b2"].CopyFrom(val2)
         state["cursor"] += 1
 
         operation.record.CopyFrom(record)
-        yield connector_sdk_pb2.UpdateResponse(operation=operation)
+        yield connector_sdk_pb2.UpdateResponse(record=record)
```

Comment:
same suggestion as above, please check wherever we are yielding an `UpdateResponse`

Reply:
I have removed operation instance.